From 41e9e6b5aa13d65480cc960a7f15f97d74d64090 Mon Sep 17 00:00:00 2001
From: Laurent Charpentier <laurent_pubs@yahoo.com>
Date: Mon, 22 Jan 2018 10:49:45 +0100
Subject: [PATCH] fixed unknown type pid_t

Fixed 'unknown type pid_t' gcc compile error in dcbtool_cmds.c and
lldptool_cmds.c

Signed-off-by: Laurent Charpentier <laurent_pubs@yahoo.com>
[Upstream status: https://github.com/intel/openlldp/pull/46]
---
 dcbtool_cmds.c  | 1 +
 lldptool_cmds.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/dcbtool_cmds.c b/dcbtool_cmds.c
index a5cd0fe..ee5c144 100644
--- a/dcbtool_cmds.c
+++ b/dcbtool_cmds.c
@@ -27,6 +27,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <ctype.h>
+#include <sys/types.h>
 #include "clif.h"
 #include "dcbtool.h"
 #include "lldp_dcbx_cmds.h"
diff --git a/lldptool_cmds.c b/lldptool_cmds.c
index daef8c8..c793e34 100644
--- a/lldptool_cmds.c
+++ b/lldptool_cmds.c
@@ -27,6 +27,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <ctype.h>
+#include <sys/types.h>
 #include "clif.h"
 #include "dcb_types.h"
 #include "lldptool.h"
-- 
2.14.3

